lib/Renard/Curie/Component/PageDrawingArea.pm

```diff
@@ -325,7 +325,7 @@ fun increment_scroll( (InstanceOf['Gtk3::Adjustment']) $current ) {
 	$current->set_value($adjustment);
 }
 
-=func decrement_scroll
+=fun decrement_scroll
```

Comment:
This should stay as `=func`. This is part of the POD directives for the documentation.